packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2219,7 +2314,7 @@ async function handler(
       );
     }
 
-    addVideoCallDataToEvt(originalRescheduledBooking.references);
+    addVideoCallDataToEvt(originalRescheduledBooking.references, evt);
```

Comment:
Shouldn't this be assigned to a variable? Like the previous use of `addVideoCallDataToEvt`